packages/@aws-cdk/yaml-cfn/lib/yaml.ts

```diff
@@ -54,6 +54,6 @@ const shortForms: yaml_types.Schema.CustomTag[] = [
 function parseYamlStrWithCfnTags(text: string): any {
   return yaml.parse(text, {
     customTags: shortForms,
-    schema: 'yaml-1.1',
+    schema: 'core',
```

Comment:
But are we sure using the `core` schema is not going to break _other_ stuff? I.e: is `"No"` going to be encoded correctly (read: quoted), or is it going to be encoded YAML-1.2-style (which in Yaml 1.1 is an alias for `false`?

Reply:
Good point! I added a test confirming that `No` is deserialized correctly (as `"No"`, not `false`).

Let me know if you want to see some more tests of some edge-cases in the YAML handling!